projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-user/user.component.html

```diff
@@ -1,9 +1,9 @@
-<div *ngIf="auth.authState| async; then authenticated else none">
+<div *ngIf="auth.currentUser !== null; then authenticated else none">
 
 </div>
 
 <ng-template #authenticated>
-  <mat-card *ngIf="auth.user | async as user">
+  <mat-card *ngIf="auth.currentUser as user">
```

Comment:
This part isn't identical in functionality since this worked perfectly out of the box and I was too lazy to figure out how to get a user stream because of that.

Similar syntax was used in multiple template files, please check/fix those manually.